### ISSUE TITLE
Token VBL removed via setTokenVBL() still affects vision

### ISSUE THREAD

**Issue by Phergus** [RPTools/maptool#485]
**Describe the bug**
Using the macro function `setTokenVBL()` to remove VBL from a token leaves "shadow" VBL behind that still affects vision until some event that causes the overall VBL topology to be updated.

**To Reproduce**
1. On a map place a token with vision.
2. Place a second token on map and using the Edit Token dialog, VBL tab, give it VBL.
3. Select or hover over token with vision to confirm 2nd token has VBL and is blocking vision.
4. Create a macro that removes token VBL per the example from [setTokenVBL()](http://www.lmwcs.com/rptools/wiki/setTokenVBL).
5. Use macro to remove VBL from 2nd token.
6. Hover over or select 1st token to see that vision is still blocked.
7. Click on VBL tool to see that no VBL is seen.
8. Edit 2nd token and observe no VBL is shown.
9. Close Edit Token dialog and observe that vision is no longer blocked for 1st token.

**Macro to Clear Token VBL**
```
[h: finalVblData = "{'shape':'none'}"]
[h: setTokenVBL(finalVblData)]
```

**Expected behavior**
Expect VBL to be updated such that no other actions are required.

**MapTool Info**
- Version: 1.5+
- Install: Any

**Desktop (please complete the following information):**
 - OS: Windows
 - Version: 10

**Additional context**
Reported on MapTool Discord by flaedda.

Demo campaign file:  
[VBLIssue.cmpgn.zip](https://github.com/RPTools/maptool/files/3248750/VBLIssue.cmpgn.zip)
(Remove ".zip" from name.)

**Comment by Phergus** [comment]
Another nuance to this issue has been found.  Clearing the VBL from the Edit Token dialog works on the client that does the clearing but this does not carry over to other clients which still show vision as blocked.

**Comment by Phergus** [comment]
Fix in 1.5.3.  Closing.
